cardano-cli/src/Cardano/CLI/EraBased/StakeAddress/Command.hs

```diff
@@ -32,12 +33,12 @@ data StakeAddressCmds era
       NetworkId
       (Maybe (File () Out))
   | StakeAddressRegistrationCertificateCmd
-      (ShelleyBasedEra era)
+      (Exp.Era era)
```

Comment:
What are the implications for this change. Is this now only `ConwayEraOnly` or `ConwayEra + 1`?

Reply:
{ConwayEra, ConwayEra + 1}